Introduction.Rmd

```diff
@@ -137,64 +159,99 @@ Make sure you have a recent version of RStudio. You can check you have the right
 
 ```{r, eval = FALSE}
 install.packages("rstudioapi")
+# TODO: update this version number?
 rstudioapi::isAvailable("0.99.149")
 ```
 
-If not, you may need to install the preview version from <http://www.rstudio.com/products/rstudio/download/preview/>. This gives you access to the latest and greatest features, and only slightly increases your chances of finding a bug. 
+If not, you may need to install the preview version from <https://www.rstudio.com/products/rstudio/download/preview/>. This gives you access to the latest and greatest features, and only slightly increases your chances of finding a bug. 
 
-Use the following code to install the development versions of devtools and usethis. This is particularly important during the development of the book.
+The following code installs the development versions of devtools and usethis, which may be important during the revision of the book.
 
 ```{r, eval = FALSE}
 devtools::install_github("r-lib/devtools")
 devtools::install_github("r-lib/usethis")
 ```
 
-You'll also need a C compiler and a few other command line tools. If you're on Windows or Mac and you don't already have them, RStudio will install them for you. Otherwise:
+You'll also need a C compiler and a few other command line tools.
```

Comment:
If you want to build packages containing C or C++ code, the topic of Chapter XYZ, you'll also need ...